doc/api/fs.md

```diff
@@ -3462,8 +3462,8 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `atime` {integer}
-* `mtime` {integer}
+* `atime` {number|string|Date}
+* `mtime` {number|string|Date}
```

Comment:
I think it would be best to keep `integer` instead of `number`. We might want to have a stricter validation here at some point. I am not sure what the behavior would be in case a decimal is used instead of an integer.

Reply:
It seems that there is no rounding, so decimal will be used as decimal.
https://github.com/nodejs/node/blob/06929956f9c438d29ca127d8be147e7969bfae35/lib/internal/fs/utils.js#L452-L457

Reply:
In that case I think number is more appropriate. Sorry for the trouble.

Reply:
Fixed in https://github.com/nodejs/node/pull/29666/commits/22157efea1c15b2078da9667fbddf1d8b6540b9c